src/bentoml/models.py

```diff
@@ -257,14 +301,19 @@ def create(
         context=context,
     )
     external_modules = [] if external_modules is None else external_modules
-    imported_modules = []
+    imported_modules: t.List[ModuleType] = []
     try:
+        res.flush()
         res.enter_cloudpickle_context(external_modules, imported_modules)
         yield res
     except Exception as e:
         raise e
+    except KeyboardInterrupt:
+        # During save, if user hit Ctrl+C, we will remove the item path cleanly
+        # so that validate() will work correctly.
+        res._fs.removetree(res.path_of(res.tag.path()))
+        raise
```

Comment:
This shouldn't have mattered; do you know why this is necessary?

Reply:
My suggestion is during the move, if users press Ctrl+C, it is not cleanup correctly.

Reply:
Are you going to remove this?

Reply:
done